packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -787,7 +790,13 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
   }
 
   private _shouldInputLoseFocus(element: HTMLInputElement, isForward?: boolean) {
-    if (element &&
+    if (this.props.tabPermission === FocusZoneTabbableElements.inputOnly && !this._processingTabKey) {
```

Comment:
You don't know if the element is an input element at this point. This needs to be moved down to the if statement below.

Reply:
I believe we should. The element we pass in is of type: HTMLInputElement

However, I think it would make more sense to add it with the rest of the cases, so I'll move it down.

Reply:
Inputs can be of different types, that's why the check below is set up the way that it is